packages/notte-browser/src/notte_browser/session.py

```diff
@@ -178,13 +178,14 @@ def previous_interaction_actions(self) -> Sequence[InteractionAction] | None:
         return actions
 
     @track_usage("local.session.replay")
-    def replay(self, screenshot_type: ScreenshotType = config.screenshot_type) -> WebpReplay:
-        observations = list(self.trajectory.observations())
-        screenshots: list[bytes] = [obs.screenshot.bytes(screenshot_type) for obs in observations]
+    def replay(self) -> WebpReplay:
+        screenshots_traj = list(self.trajectory.all_screenshots())
```

Comment:
maybe add a test to check that the 3 screenshots are different but othewise LGTM